sale_stock_info_popup/static/description/index.html

```diff
@@ -3,7 +3,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-<meta name="generator" content="Docutils 0.15.1: http://docutils.sourceforge.net/" />
+<meta name="generator" content="Docutils 0.16: http://docutils.sourceforge.net/" />
```

Comment:
Please pin your docutils version locally to the same one.

Reply:
I recommend you to install the tool with pipx, you'll avoid these kind of problems:

```bash
pipx install https://github.com/OCA/maintainer-tools/archive/master.zip
```